source/stylesheets/scss/furatto/_mixins.scss

```diff
@@ -0,0 +1,24 @@
+//Mixins to define the visibility media queries support
+@mixin responsive-invisibility {
+  display: none !important;
+
+  tr, th, td {
+    display: none !important;
```

Comment:
!important should not be used<br>Each selector in a comma sequence should be on its own line